### PR TITLE
New manual website

### PR DESCRIPTION
**Pull request by lucacasonato**
Towards #9

Design constraints (TODOs):
- [x] Run on Deno for local development
- [ ] Run on Deno Deploy for production
- [x] Server side rendered
- [x] Support a light and dark theme
- [ ] OpenGraph metadata
- [x] High quality markdown rendering with a good markdown stylesheet
  - Uses GitHub light & dark stylesheet
  - [x] Switch to [deno-gfm](https://deno.land/x/gfm)
  - [x] Links on headings
  - [x] Syntax highlighting
- [x] Edit on GitHub link
- [ ] Next + Previous link
- [x] Content warning on preview URLs  
- [x] Sidebar
- [ ] Version selector
- [ ] Accessible & easy to use on mobile and desktop
- [x] 100/100/100/100 Lighthouse score
- [ ] Integrate Algolia DocSearch like the current site does
- [ ] Test coverage
  - [x] Unit tests
    - There is very good unit test coverage.
  - [ ] Integration tests
  - [ ] Run tests in CI

### CODE REVIEW

File: www/versions.json [hunk 1]
```diff
@@ -0,0 +1,257 @@
+{
+  "std": [
+    "0.120.0",
+    "0.119.0",
+    "0.118.0",
```
Comment (bartlomieju) [inline www/versions.json:5]: @lucacasonato are we gonna have to sync this list in both `manual` and `dotland` repos?

**Comment by littledivy** [comment]
@lucacasonato `gfm@0.1.17` removed styling for elements outside markdown body. https://github.com/lucacasonato/deno-gfm/commit/e8c376a1508b5c2ecd832b887d7d63b05615be52#diff-5b62c90de030e6b2f6bf796148061753e6df05e47df1c926725228b84c371e82 and older version of gfm don't work with Deno 1.17

**Comment by UltiRequiem** [comment]
I would love updates on this PR.

**Comment by ry** [comment]
Manual is now rendered as part of https://github.com/denoland/dotland so this PR isn't necessary anymore.

**Pull request closed without merging**